grease-cli/src/Grease/Main.hs

```diff
@@ -487,6 +492,28 @@ interestingConcretizedShapes names initArgs (ConcArgs cArgs) =
     names
     (Ctx.zipWith (\s s' -> Const (Maybe.isJust (testEquality s s'))) cShapes initArgs')
 
+-- | Helper, not exported
+--
+-- Initialize the symbolic file system.
+initialLlvmFileSystem ::
+  ( C.IsSymInterface sym
+  , Mem.HasPtrWidth ptrW
+  ) =>
+  C.HandleAllocator ->
+  sym ->
+  C.NatRepr ptrW ->
```

Comment:
Why not use `?ptrWidth` from the `HasPtrWidth` constraint?

Reply:
This was a leftover from the type signature for `crucible-llvm`'s [`initialLLVMFileSystem`](https://github.com/GaloisInc/crucible/blob/9cb4f93d17c48890a1643a005120b365e6873c56/crucible-llvm/src/Lang/Crucible/LLVM/SymIO.hs#L180-L183) function, which has both a `HasPtrWidth` constraint and an explicit `NatRepr` argument. You're absolute right, though–it's redundant to have both, and I'm not sure why `crucible-llvm` does things this way. I've removed the `NatRepr` argument from the `grease-cli` function that calls `initialLLVMFileSystem`.

Reply:
https://github.com/GaloisInc/crucible/issues/1394